plugins/csi/client.go

```diff
@@ -881,7 +881,7 @@ func (c *client) NodePublishVolume(ctx context.Context, req *NodePublishVolumeRe
 				"volume %q is already published at target path %q but with capabilities or a read_only setting incompatible with this request: %v",
 				req.ExternalID, req.TargetPath, err)
 		case codes.FailedPrecondition:
-			err = fmt.Errorf("volume %q is already published on another node and does not have MULTI_NODE volume capability: %v",
+			err = fmt.Errorf("volume %q does not have MULTI_NODE volume capability: %v",
```

Comment:
these two are different from `ControllerPublishVolume` now, but they're still the same as each other?

Reply:
that makes sense. I suppose I figured while we're differentiating error messages that these could be made different too, but it does seem less important than distinguishing between controller plugin activity vs node plugin activity. :+1: